streams/src/test/java/org/apache/kafka/streams/processor/internals/InternalTopologyBuilderTest.java

```diff
@@ -554,15 +555,16 @@ public void shouldAddInternalTopicConfigWithCleanupPolicyDeleteForInternalTopics
         builder.setApplicationId("appId");
         builder.addInternalTopic("foo");
         builder.addSource(null, "source", null, null, null, "foo");
-        final TopologyBuilder.TopicsInfo topicsInfo = builder.topicGroups().values().iterator().next();
+        final InternalTopologyBuilder.TopicsInfo topicsInfo = builder.topicGroups().values().iterator().next();
         final InternalTopicConfig topicConfig = topicsInfo.repartitionSourceTopics.get("appId-foo");
         final Properties properties = topicConfig.toProperties(0);
         assertEquals("appId-foo", topicConfig.name());
         assertEquals("delete", properties.getProperty(InternalTopicManager.CLEANUP_POLICY_PROP));
         assertEquals(1, properties.size());
     }
 
-    @Test(expected = TopologyBuilderException.class)
+    @SuppressWarnings("deprecation")
+    @Test
```

Comment:
No. It's about usage of `TopologyBuilderException`. Again, related to https://issues.apache.org/jira/browse/KAFKA-5660